tests/integration/plugins/test_rust.py

```diff
@@ -29,33 +29,11 @@ def test_rust_plugin(new_dir):
         parts:
           foo:
             plugin: rust
-            source: .
+            source: simple
         """
     )
     parts = yaml.safe_load(parts_yaml)
 
-    Path("Cargo.toml").write_text(
```

Comment:
all the files in the `test_rust/` dir have the contents that were removed from these tests, verbatim; to generate them I ran the original tests and then just copied the contents from the temp dir.